### DISCUSSION TITLE
GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance.

### DISCUSSION THREAD

**Discussion by johnsy616** [community/community#143194] [Actions]
### Select Topic Area

Question

### Body

I have removed everything that might violate the TOS. could you help to check whether you can recover the Actions for my account.
Thank you.

**Comment by samus-aran** [comment]
Hey @johnsy616 👋

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by johnsy616** [reply]
got it, thanks
